### PR TITLE
ci: Reduce expiry time for GitLab artifacts

### PR DESCRIPTION
**Pull request by paulgessinger**
Also removes trailing whitespace in the gitlab configuration.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -77,6 +77,7 @@ build_exatrkx:
     exclude:
       - build/**/*.o
       - build/bin/ActsIntegrationTest*
+    expire_in: 6 hours
```
Comment (AJPfleger) [inline .gitlab-ci.yml:80]: Could we keep this to maybe 1 day or at least 12 hours?
The GitLab jobs often take quite long to start, so I let them frequently run over night. But then the artifacts would be gone in the morning.

**Review by AJPfleger — COMMENTED** [review]
Why? Did we run out of space?

**Comment by paulgessinger** [comment]
We were using a lot of storage

**Review by AJPfleger — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :red_circle: Athena integration test results [1a2deaf20ba5a75981e72a8d281d5be506d7e150]
### Build job with this PR failed!
Please investigate the [build job](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/34718959) for the [pipeline](https://gitlab.cern.ch/acts/acts-athena-ci/-/pipelines/6615946)!

**Pull request merged**